bus-arrival/index.html

```diff
@@ -1,6 +1,8 @@
ZugfgpoKTyOAeUFQd/qi7rEOCbBrANmyF2uijNqY9K1J2K+ENzWZNA/zSAXYpfSbCGQaTeNU0QKWRnlnB8CBoACr8G/nDsAmPLVRROY0C89bkL832cWRLYDBoA6haCftmTZMtVFK7gX3A+dUyYVSp7Z4MzQQNA2VKwq4uftitmzwAN/w5vvS2UwKPq7SLTHJzl6NNcF6SVnQxypwHLNQqk8K8Vn1ueaNIR0dFg4kWwhu+0qtPBLls2MtqleL0L6PgPil8tcBt4s/VmwABQtCFkyrGjjLYhVvUAG/9JcQQuI28TkZkNFgBqtoR57L3FaBvjVBNsYGDideDWzDfVCBYAajaFep3azmhHYzQvsIr/SE/DHxr39oT1eaAAULItfNaebYx2IT5zg1X8Vz1vRtEBlxeBAkDpqQCX80lG2x6b/oAa/qtPMNGaOe+zlgoSAIoXAie2Mh/b4tIjVOLoI8uR9N7bBMqH4NLhAQJAwcWhREGIlIxLn2AZabyzokt3jjtzjvwdBAgABZeHCyQeMLYcRcE80r4CWGuSJ/vl+Jxe8pItHwDqbRAhlLjE2FIUBZ8j6Z3m7vfw8uR8/bnfVu6ddAAot0UMYfMG5udAHFaNFZDSyv361HsUcCx3Iegn3gU37S1HoexHVwAo3Q4iKApTdsdg5Si5TZxuwi/a2PEvBA3ccePW0VHAHsJEWrVt4iSKwrQdL9UfBl5Jlvkqz1M+e8lrjneJMH0xot0Bt7RqG0X6F4WXYBj4Ir1TvNXMZ9Et26iBS4Eq3KXfGt7wamS8n1Fsq1hfp86y+A8DK0H+6pX2J9tI4XfcsL+a4hcKXoAZwy+2/us9wcmvdkvglVZrs2gJpySeATtuqD0MiLeLz1swX6XWTWs+VWWnD/No1rBbszSYq63WdvEy9h5l/g6cBJXlUcTogryc4bk4EzQS3/LYVAEmjm1k/nYeBoX1UCi7CtLq6JFqQxD9lmJHxkjasoHFfBgwOVUIsv6+oMQ/g7xOUbFDo6RtvsQkXFF4GMgjwRiBjI/oEvzYoP4LFTcKlp8XiPcw0EGS7R8CuToXhE9TA3/phnoHRwaR2MUk7Lj4BNSUKyKtVQXSShmFym3wYdazCh4dG8zWJJOw+zioqYF+Wu0ciNQGnL52BIRMXcnDo4M6v43JuKJmt6DU8fFrq0OYVepInB9v1Edz4ydXHeio5vHxge3ZzmRs26XisgHNQCn6p85KvwK/WaPna0WUk8qP+yV34KQz43wKZRhqrAmSLwrSvrF3709JRFEcwO+5ugkbIA/FFgSlh5gKiFhgSOYz1EzUzAcqmmhqajNJRcWYvd/1S39wzjSVYyK7sMuehf38zk/L3nP2e+7d7XYRfDJUAGtkN7j2cjcSrabCVEdHNteWP37aeLm7LuC3GaIMh6EgP13XCTajUYpWFMepUH6hID+mfnR1IEnRek8UQ9cKPE37CC4DVoqUdYAoB1NjAJ76kJ0f26NI7RFFqesEnjx6VHWgx0tR8qIeA5zgUivwZQnYCB7LFKVlojTMkAH4ujmJ5yh5A8olwNtAlMdnAd4cE2j2DKJ8EMB3HICP263An+kqknawPULRieAfA2UpA2bgz3kWRzIUo+igOhMuWRkAMPeiGBKlKDIpoly33SCEuc9PZPcYWRpkVWIH+BczaQZBWraJ3PYpKkqZAmXTaAFhqvQ6IqvhNEUkjfYwAF/NYRDI1HuDITJ6jigM8D4niseEOkGo6YCcycAORWOHlAKd1gBCGVvmF4hMLqxRJN4qYx9QbpftIJyzq8NGZDFwhaIwophtIDkxSw7IgynexJIiY5vipoNnFIFSaAD+ae6FvHhaloqYEV5favHAoQ8UgTektDRWQZ6mNXUskRxbp5mGP15R2W2USgPw11jAA/ky1ev9DJEM49fXm+CoL1RmqZK7/odW41AAh3vIZyOis/mG3A44busHldWiQmeAudR1Q0GMdm1HJUNEwlR2aO1GONHWayqjtJK2gQrCDjmhUJ47U/pGHSmIrlE/dccDp/j8lcomougRUA6VFQYQw/l6jd7FsUQglnPpNfW1kNv3b1Qm67heCy46rs8MYjFY7PHEbJOPa2bIKZhmztc0m4jbLQbg7UCmf8A6gg+ESqxyyghiMzi67eF475RWk6iZvK/X35+sSWi0U73xsL3bYYB8fJelCkRK/P7/bVXrBPw+y9AJpku5/h91O+EB9Ir/NLhYsv3//67VOAC7rSInQqkSff7PYqG/E7DLnQqr+W8BWH0tIPehaLNB75uyu/6H2JluwO2gSK3gSEnNfwVg5u2A2lZRysDb0tn/IZxfi7sflL4MeHfKcfk/whZ6Cmg9Df1MU0mly3X5P4rT3ASEbmo4QshwxkolY80ofv+/ONiOFmQBobOlgyW/NaSoRFLlEv7xoQvVGwEJY31ogRwRi1AJRJR7/lciupmwGWRnDt/TkWPak14qMm+yvLI/nq7NuI0gI6N75ho5ScOyV9TLv6yu/tks3G07D7I433Z3gWTVs2elIrHuKe39X8XGBdxOKCqnO8CRHAaSUSqC6PsyGvzlz+Y6e8sARWG4ddZlI3yMZiK0QJFMOQd/AulcE2dMICnTmQmXjvB3YWUjSvMWfbJS5rmfcGPvAl0WkISlK3B5jAjW/jBYTfNQHXyoNv55uuTq77tlBNEYq/r6XZdI3noeBa1UEGvwkdr4FWjMH7oarjVAQQy14ash/xgp2PDK/qaX8uLd3F9RI1+xsOfGZxMV9k4QqNNekZgdP8cSEbUPJoMjp5aD6pFgclBd+KVg48ZDgYm2itaqWhNkYaqtaq1omwiExjmWSGV4LvZiY/HKsZJgvbK48SI2p974RcE2r1ZyD7Z9da6m+fkmV51v+wFXudrMkmK6MNozd3EwFhu8ONczqvb6KpVKpVKpVCqVSqX61R4cCAAAAAAI8rce5AoAAAAAAAAAAAAAAAAAAAAAgLcARF1l7qb6M5cAAAAASUVORK5CYII=">
```

Comment:
First, can get rid of the `-precomposed` part because it's no longer needed since iOS 7.

Second, can link to the original icon image here `icons/icon-192.png` instead of this data URI?

Reply:
:+1:  implemented in https://github.com/cheeaun/busrouter-sg/commit/fd739242ffd5e83af6bd7b393727e9b3f5aac6ff